frontend/src/graphql-components/query/instances.query.graphql

```diff
@@ -7,6 +8,7 @@ query ownedInstances($tenantNamespace: String!) {
         name
         namespace
         creationTimestamp
+        labels
```

Comment:
```suggestion
        labels {
          workspaceName
        }
```
Something alike could actually even work (clearly with the right key, I don't remember which it is now), would you mind trying it out?

Reply:
I tried and it doesn't work

Reply:
`labels` field contains a `JSON` object, not a graphQL type. So, you should parse the given JSON and work on it.

Reply:
but, could be better adding to the query the same fields about workspace that are in Instances Label selector query? so that the response should be a graphql type and not a JSON

Reply:
Pay attention, valid label keys have two segments: an optional prefix and name, separated by a slash `/` (see [labels and selector](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)). So, in this case, checking labels with kubectl you should see something like that: `"crownlabs.polito.it/workspace: sid"`, but the output given by GraphQL will be something like this `"crownlabsPolitoItWorkspace": "sid"`.  I don't know the reason, but `.` and `/` are removed and the words are written in camelCase.

Reply:
I solved it, it returns an object and it's not necessary to parse it, now I'm casting it with a function to an explicit type.

Reply:
Another problem was that I didn't add the label field to the subscriptions. Now it seems to be ok